spotbugs/src/gui/edu/umd/cs/findbugs/gui2/DisplayNonmodelMessage.java

```diff
@@ -109,42 +109,26 @@ static void sleep(int seconds) {
     }
 
     /*
-    static Runnable moveToFrontLater = new Runnable() {
-        @Override
-        public void run() {
-            sleep(5);
-            SwingUtilities.invokeLater(moveToFront);
-        }
+    static Runnable moveToFront = () -> {
+        JFrame frame = messageFrame;
+        if (frame != null)
+            frame.toFront();
     };
-     */
 
-    static Runnable clearAlwaysOnTopLater = new Runnable() {
-        @Override
-        public void run() {
-            sleep(5);
-            SwingUtilities.invokeLater(clearAlwaysOnTop);
-        }
+    static Runnable moveToFrontLater = () -> {
+        sleep(5);
+        SwingUtilities.invokeLater(moveToFront);
     };
+    */
 
-    /*
-    static Runnable moveToFront = new Runnable() {
-        @Override
-        public void run() {
-            JFrame frame = messageFrame;
-            if (frame != null) {
-                frame.toFront();
-            }
-        }
+    static Runnable clearAlwaysOnTop = () -> {
+        JFrame frame = messageFrame;
+        if (frame != null)
```

Comment:
please always use {} braces for ALL if/else/for/while etc constructs.

Reply:
I was not aware of that rule/recommendation => will be fulfilled from now on.